backend/src/akvo/lumen/lib/public.clj

```diff
@@ -164,16 +67,15 @@
 
 (defn visualisation-response-data [tenant-conn id config]
   (try
-    (let [[tag vis] (visualisation/fetch tenant-conn id)
-          ;; provisional condition until akvo.lumen.lib.visualisation.maps is keywordized, tested and speced too
-          vis (if (= "map" (:visualisationType vis))
-                vis
-                (update vis :spec keywordize-keys))]
-     (when (= tag ::lib/ok)
-       (visualisation tenant-conn vis config)))
-    (catch Exception e (do
-                         (log/warn e ::visualisation-response-data (str "problems fetching this vis-id: " id))
-                         nil))))
+    (let [vis-types #{"pivot table" "line" "bubble" "area" "pie" "donut" "polararea" "bar" "scatter" "map"}
```

Comment:
This set is duplicated from line 24. 

Also, probably `map` should not be there.

Reply:
not really, in line 24 there is a map instead of a set

Reply:
The set has (or should) have the same keys than the map in line 24, hence, it is duplicated.